src/common/navigation.yaml

```diff
@@ -45,32 +71,6 @@ sections:
           - title: Tutorials
             pages:
               - path: /integrate-indexing
-      - path: /stacks-blockchain
```

Comment:
all these "old" URLs need to be matched with new redirects - would double check that